optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -37,36 +44,35 @@ export const GraphHistory: FC<{
   const [filterCompleteTrial, setFilterCompleteTrial] = useState<boolean>(false)
   const [filterPrunedTrial, setFilterPrunedTrial] = useState<boolean>(false)
 
-  const [targets, selected, setTarget] = useObjectiveAndUserAttrTargets(study)
-  const trials = useFilteredTrials(
-    study,
+  const [targets, selected, setTarget] = useObjectiveAndUserAttrTargets(
+    studies.length !== 0 ? studies[0] : null
+  )
```

Comment:
`Target` must be prepared by passing all studies, not only the first study. Otherwise, we cannot display the user attributes correctly. Could you also update the `useObjectiveAndUserAttrTargets`?